.changeset/heavy-toes-attend.md

```diff
@@ -0,0 +1,8 @@
+---
+"@latticexyz/world-modules": patch
+"@latticexyz/store": patch
+"@latticexyz/world": patch
+"@latticexyz/cli": patch
+---
+
+Upgraded the EJS package to 3.1.10, and removed it from `store`, `world`, and `cli`.
```

Comment:
this might read better in the changelog if we add the "upgrade" changeset to the corresponding packages and "removed" from the others

I try to think of changesets in the context of a single package and how it'll look as if I was reading just that package's changelog (aside from the unique way we roll them up to a single changelog)

Reply:
I like this! They are kind of unrelated changes too.